packages/shared/src/components/cards/PostList.tsx

```diff
@@ -84,7 +84,7 @@ export const PostList = forwardRef(function PostList(
     return (
       <div className={`relative ${styles.cardContainer}`}>
         {card}
-        <TrendingFlag trending={trending} listMode />
+        <TrendingFlag trending={trending} />
```

Comment:
I think this should be in list mode.

```suggestion
        <TrendingFlag trending={trending} listMode />
```

Reply:
This TrendingFlag wasn't in listMode before i worked on it so just checking on this as seems this would have a change somewhere if i'm not trying to change any ui just add a new component? @sshanzel